src/api/tunnels.rs

```diff
@@ -44,6 +51,14 @@ pub struct CreateCommand {
     /// The length of time in seconds for the tunnel to live.
     #[arg(long)]
     ttl: Option<u16>,
+
+    /// Whether or not to wait for a state other than "requested"
+    #[arg(long)]
+    wait: Option<bool>,
+
+    /// The number of attempts to be used with "--wait". Max: 5, Min: 1
+    #[arg(long)]
```

Comment:
I think this line could be written differently to specify the min/max values programmatically so that 1) you don't have to manually write the docs, and 2) so you don't have to "clamp" later in this PR. I think, confirm with @andresperi if confused.